Documentation/subcommands/run.md

```diff
@@ -309,32 +309,32 @@ If you don't use systemd, you can use [daemon](http://www.libslack.org/daemon/)
+| `--dns` | none | IP Address | Name server to write in `/etc/resolv.conf`. It can be specified several times |
+| `--dns-opt` |  none | DNS option  | DNS option from in resolv.conf(5) to write in `/etc/resolv.conf`. It can be specified several times. |
+| `--dns-search` |  none | Domain name | DNS search domain to write in `/etc/resolv.conf`. It can be specified several times. |
+| `--exec` | none | Path to executable | Override the exec command for the preceding image. |
+| `--inherit-env` |  `false` | `true` or `false` | Inherit all environment variables not set by apps. |
+| `--interactive` |  `false` | `true` or `false` | Run pod interactively. If true, only one image may be supplied. |
+| `--mds-register` |  `false` | `true` or `false` | Register pod with metadata service. It needs network connectivity to the host (`--net` as `default`, `default-restricted`, or `host`). |
+| `--memory` |  none | Memory units (ex. `--memory=50M`) | Memory limit for the preceding image in [Kubernetes resource model](http://kubernetes.io/v1.1/docs/design/resources.html) format. |
+| `--mount` |  none | Mount syntax (ex. `volume=NAME,target=PATH`) | Mount point binding a volume to a path within an app. See [Mounting Volumes without Mount Points](#mounting-volumes-without-mount-points). |
+| `--net` |  `default` | A comma-separated list of networks. (ex. `--net[=n[:args], ...]`) | Configure the pod's networking. Optionally, pass a list of user-configured networks to load and set arguments to pass to each network, respectively. |
+| `--no-overlay` |  `false` | `true` or `false` | Disable the overlay filesystem. |
+| `--no-store` |  `false` | `true` or `false` | Fetch images, ignoring the local store. See [image fetching behavior](../image-fetching-behavior.html) |
+| `--pod-manifest` |  none | A path | The path to the pod manifest. If it's non-empty, then only `--net`, `--no-overlay` and `--interactive` will have effect. |
+| `--port` |  none | A port number (ex. `--port=NAME:HOSTPORT`) | Ports to expose on the host (requires [contained network](https://github.com/coreos/rkt/blob/master/Documentation/networking.md#contained-mode)). |
+| `--private-users` |  `false` | `true` or `false` | Run within user namespaces (experimental). |
+| `--set-env` |  none | An environment variable (ex. `NAME=VALUE`) | An environment variable to set for apps. |
+| `--signature` |  none | A file path | Local signature file to use in validating the preceding image |
+| `--stage1-url` |  none | URL with protocol | A URL to a stage1 image. HTTP/HTTPS/File/Docker URLs are supported. |
+| `--stage1-path` |  none | Absolute or relative path | A path to a stage1 image. Absolute and relative paths are supported. |
+| `--stage1-name` |  none | Image name | A name of a stage1 image. Will perform a discovery if the image is not in the store. |
```

Comment:
Should we give an example of image name? I think it could be confusing: I tried `--stage1-name=coreos` or `--stage1-name=fly` but it was not working. In fact, the syntax would be `--stage1-name=coreos.com/rkt/stage1-coreos`.